contracts/mocks/MockBosonRouter.sol

```diff
@@ -169,7 +169,7 @@ contract MockBosonRouter is
         whenNotPaused
     {
         checkLimits(_metadata, address(0), address(0), 0);
-        requestCreateOrder(_metadata, 5, address(0), address(0), 0);
+        requestCreateOrder(_metadata, ETHETH, address(0), address(0), 0);
```

Comment:
Is this intentional?

Reply:
This is intentional. The` 5` has to be in there in order to make one of the test cases fail. It had to be added back in  anytime BosonRouter.sol changes are copied into MockBosonRouter.sol. There were too many changes in BosonRouter to change them one by one in MockBosonRouter.sol and expect to get them right.